autogpt_platform/backend/.env.default

```
@@ -75,6 +75,20 @@ V0_API_KEY=
+# AutoPilot — self-hosted LLM (Ollama et al.)
+# Set CHAT_USE_LOCAL=true to route AutoPilot through an OpenAI-compatible
+# endpoint instead of OpenRouter / Anthropic. CHAT_BASE_URL must point at
+# the local server (e.g. http://host.docker.internal:11434/v1 for Ollama
+# running on the Docker host) and CHAT_API_KEY can be any non-empty string
+# — Ollama doesn't validate it. The CHAT_*_MODEL fields must be set to
+# bare model names served by the local backend (e.g. llama3.2:3b);
+# OpenRouter-style provider/model slugs do not resolve. Extended-thinking
+# mode is automatically downgraded to fast under this transport because
+# the Claude Agent SDK CLI requires Anthropic's wire protocol.
+CHAT_USE_LOCAL=false
+CHAT_BASE_URL=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CHAT_BASE_URL key should go before the CHAT_USE_LOCAL key

(UnorderedKey)